index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+</head>
+
+<body>
+    <nav>
+        <img src="https://pmcvariety.files.wordpress.com/2017/09/youtube-logo.png?w=896" width="200" height="100" alt="" width="200" height="100">
+        <input class="searchbar" type="search">
+        <input class="searchbutton" type="button" value="search">
+        <div class="right">
+            <img src="https://via.placeholder.com/35x35/fff" alt="">
+            <img src="https://via.placeholder.com/35x35/000" alt="">
+            <img src="https://via.placeholder.com/35x35/fff000" alt="">
+        </div>
+    </nav>
```

Comment:
The navbar seems invisible. The content is present but the styling is not yet up to the task. You can use developer tools to inspect the page so you can get the exact image used. [Here are some tips and tricks to use developer tools](https://www.keycdn.com/blog/chrome-devtools). Please take your time to make it as close as possible to the original image as shown in the image below:

**Original**
![youtube](https://user-images.githubusercontent.com/31304943/69574008-77245600-0fc7-11ea-8452-e4df8881b83f.png)

**Your design**
![tube](https://user-images.githubusercontent.com/31304943/69574134-c1a5d280-0fc7-11ea-952d-cd1b8ae6a9a8.png)